src/claim/claim-link-object-identity.js

```diff
@@ -42,6 +42,9 @@ export class LinkObjectIdentity {
    * @param {Buffer} auxData - Auxiliary data to complement hash object
    */
   constructor(objectType: number, objectIndex: number, id: string, objectHash: Buffer, auxData: Buffer) {
+    if (objectHash.length > 31 || auxData.length > 31) {
```

Comment:
Not needed with last hash claim input definition :)

Reply:
done!